### DISCUSSION TITLE
sidebar example

### DISCUSSION THREAD

**Discussion by justinasmussen** [Megabit/Blazorise#2296] [General]
where can I find an example of the  [SideBar](https://blazorise.com/docs/extensions/sidebar/)

**Comment by stsrki** [comment]
The example can be found in the documentation on the same page you provided: https://blazorise.com/docs/extensions/sidebar/#manual

But I would recommend you to use the `Bar` component instead because it has much more features and is actively developed. The `Sidebar` is an old component and is obsolete now. I mean, it works but is replaced with the Bar.

- docs: https://blazorise.com/docs/components/bar/
- demo: https://github.com/stsrki/Blazorise/blob/master/Demos/Blazorise.Demo/Layouts/MainLayout.razor
